llm_assistant/discord_bot/ai/qa.py

```diff
@@ -0,0 +1,20 @@
+import os
+import dotenv
+from openai import AsyncOpenAI
+
+import time
+
+dotenv.load_dotenv()
+
+
+async def async_answer_question(model, question):
+  start = time.time()
+  ai_server_url = os.getenv("AI_SERVER_URL")
+  client = AsyncOpenAI(base_url=ai_server_url, api_key="FAKE")
```

Comment:
Why AsyncOpenAI here vs OpenAI down ollama/openai_chat.py

Reply:
fixed: reuse the function in `qa.py` now